### PR TITLE
LPS-133206  Resource integration

### PR DESCRIPTION
**Pull request by victortrajano**

### CODE REVIEW

File: modules/apps/digital-signature/digital-signature-web/src/main/java/com/liferay/digital/signature/web/internal/portlet/action/GetDSDocumentsAsBytesMVCResourceCommand.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+ */
+@Component(
+	immediate = true,
+	property = {
+		"javax.portlet.name=" + DigitalSignaturePortletKeys.DIGITAL_SIGNATURE,
+		"mvc.command.name=/digital_signature/get_ds_documents_as_bytes"
+	},
+	service = MVCResourceCommand.class
+)
+public class GetDSDocumentsAsBytesMVCResourceCommand
```
[the remark this answers is not in the document]
Comment (kevenleone) [reply modules/apps/digital-signature/digital-signature-web/src/main/java/com/liferay/digital/signature/web/internal/portlet/action/GetDSDocumentsAsBytesMVCResourceCommand.java:47]: Here some example with Download File

DownloadCommerceVirtualOrderItemMVCResourceCommand.java (47 and 51)
DownloadEntriesMVCResourceCommand.java (67, 72)
DownloadCertificateMVCResourceCommand.java (41, 45)

I see we have others prefixes, along Download the Liferay-portal  like Export, Get.

**Review by kevenleone — COMMENTED** [review]
In general I approve the changes, i left some comments @joseabelenda , what you think?

File: modules/apps/digital-signature/digital-signature-web/src/main/java/com/liferay/digital/signature/web/internal/portlet/action/GetDSDocumentsAsBytesMVCResourceCommand.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (joseabelenda) [reply modules/apps/digital-signature/digital-signature-web/src/main/java/com/liferay/digital/signature/web/internal/portlet/action/GetDSDocumentsAsBytesMVCResourceCommand.java:47]: That look is good.
We tried to follow the pattern of Digital Signature nomenclature.

**Review by joseabelenda — APPROVED** [review]
LGTM

**Comment by kevenleone** [comment]
resent: #19

**Pull request closed without merging**